src/hooks/useProfile.ts

```diff
@@ -2,9 +2,14 @@ import { useQuery } from '@tanstack/react-query'
 import profiles from 'decentraland-gatsby/dist/utils/loader/profile'
 
 export default function useProfile(address?: string | null) {
+  const fetchProfile = async () => {
+    if (!address) return null
+    return profiles.load(address)
+  }
   const { data: profile } = useQuery({
-    queryKey: [`user-${address}-profile`],
-    queryFn: () => (address ? profiles.load(address) : null),
+    queryKey: [`user-${address?.toLowerCase()}-profile`],
+    queryFn: () => fetchProfile(),
+    staleTime: 3.6e6, // 1 hour
```

Comment:
You can move this to the `hooks/constants.ts` file to reuse as `ONE_HOUR_STALE_TIME`